extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -1,27 +1,30 @@
-import { SOPClassHandlerName, SOPClassHandlerId } from './id';
 import { utils, classes, DisplaySetService, Types } from '@ohif/core';
-import addDICOMSRDisplayAnnotation from './utils/addDICOMSRDisplayAnnotation';
+import addSRAnnotation from './utils/addSRAnnotation';
 import isRehydratable from './utils/isRehydratable';
-import { adaptersSR } from '@cornerstonejs/adapters';
-import CodeNameCodeSequenceValues from './constants/CodeNameCodeSequenceValues';
+import { SOPClassHandlerName, SOPClassHandlerId } from './id';
+import {
+  CodeNameCodeSequenceValues,
+  CodingSchemeDesignators,
+  CORNERSTONE_FREETEXT_CODE_VALUE,
+} from './enums';
 
 type InstanceMetadata = Types.InstanceMetadata;
 
-const { CodeScheme: Cornerstone3DCodeScheme } = adaptersSR.Cornerstone3D;
-
 const { ImageSet, MetadataProvider: metadataProvider } = classes;
 
-// TODO ->
-// Add SR thumbnail
-// Make viewport
-// Get stacks from referenced displayInstanceUID and load into wrapped CornerStone viewport.
+/**
+ * TODO
+ * - [ ] Add SR thumbnail
+ * - [ ] Make viewport
+ * - [ ] Get stacks from referenced displayInstanceUID and load into wrapped CornerStone viewport
+ */
 
 const sopClassUids = [
-  '1.2.840.10008.5.1.4.1.1.88.11', // BASIC TEXT SR
-  '1.2.840.10008.5.1.4.1.1.88.22', // ENHANCED SR
-  '1.2.840.10008.5.1.4.1.1.88.33', // COMPREHENSIVE SR
-  '1.2.840.10008.5.1.4.1.1.88.34', // Comprehensive 3D SR
-  // '1.2.840.10008.5.1.4.1.1.88.50', // Mammography CAD SR
+  '1.2.840.10008.5.1.4.1.1.88.11' /** BASIC_TEXT_SR */,
```

Comment:
Please import these from @ohif/core sopClassDictionary (you will need to export it at the top level I believe)